website/docs/cloud-docs/users-teams-organizations/2fa.mdx

```diff
@@ -34,3 +34,11 @@ If you are an organization owner you can require all users within your organizat
 Click **Settings** in your organization to reach your organization'a settings page, then click **Authentication**.
 
 Click the button "Require two-factor". Please remember that all organization owners must have two-factor authentication on before this can be set.
+
+## Requiring Two-factor Authentication for Users with HashiCorp Cloud Platform 
+
+When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity:
```

Comment:
```suggestion
When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity of the user who is trying to log in:
```

Reply:
I'm still a little confused about this. Do we mean that admins have to determine all the login methods used in their org and configure 2FA separately for each login method? If so, where can they find the list of login methods?

Reply:
this enforces that a user who needs access to the org must configure 2FA.  They are not allowed to access the org unless 2FA is configured.  Regarding HCP identity, the means of configuring 2FA differs depending on the identity chosen.

For HCP Email, they must configure HCP MFA
For HCP GitHub, they must configure GitHub MFA.